specification/storage/resource-manager/readme.md

```diff
@@ -103,4 +103,70 @@ csharp:
   payload-flattening-threshold: 2
   output-folder: $(csharp-sdks-folder)/Storage/Management.Storage/Generated
   clear-output-folder: true
+```
+
+
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+``` yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  namespace: storage
+  clear-output-folder: true
+```
+
+### Tag: package-2017-06 and go
+
+These settings apply only when `--tag=package-2017-06 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2017-06' && $(go)
+output-folder: $(go-sdk-folder)/services/storage/mgmt/2017-06-01/storage
+```
+
+### Tag: package-2016-12 and go
+
+These settings apply only when `--tag=package-2016-12 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2016-12' && $(go)
+output-folder: $(go-sdk-folder)/services/storage/mgmt/2016-12-01/storage
+```
+
+### Tag: package-2016-05 and go
+
+These settings apply only when `--tag=package-2016-05 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2016-05' && $(go)
+output-folder: $(go-sdk-folder)/services/storage/mgmt/2016-05-01/storage
+```
+
+### Tag: package-2016-01 and go
+
+These settings apply only when `--tag=package-2016-01 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2016-01' && $(go)
+output-folder: $(go-sdk-folder)/services/storage/mgmt/2016-01-01/storage
+```
+
+### Tag: package-2015-06 and go
+
+These settings apply only when `--tag=package-2015-06 --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2015-06' && $(go)
+output-folder: $(go-sdk-folder)/services/storage/mgmt/2015-06-15/storage
+```
+
+### Tag: package-2015-05-preview and go
+
+These settings apply only when `--tag=package-2015-05-preview --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.

```

Comment:
Should this line be added to all tags (it is missing in some other services)?

Reply:
:( Yes, I suppose so

Reply:
Done